packages/starters/backend/sirius-components-task-starter/src/main/java/org/eclipse/sirius/components/task/starter/configuration/view/ViewsStereotypeDescriptionRegistryConfigurer.java

```diff
@@ -85,7 +85,7 @@ private View createDeckView() {
         resourceSet.eAdapters().add(new ECrossReferenceAdapter());
 
         View view = ViewFactory.eINSTANCE.createView();
-        new ViewDeckDescriptionBuilder().addRepresentationDescription(view);
+        new ViewDeckDescriptionBuilder().addRepresentationDescriptions(view);
```

Comment:
Could you please rename GANTT_VIEW_LABEL and DECK_VIEW_LABEL so that they reflect the fact that they are examples based on task

Reply:
done